src/SwatchWithForm.tsx

```diff
@@ -21,6 +22,7 @@ function SwatchWithForm({swatchConfig, setSwatchConfig, staggerType, showExperim
         setSwatchData={setSwatchConfig}
         staggerType={staggerType}
         showExperimentalFeatures={!!showExperimentalFeatures}
+        formClasses={formClasses}
```

Comment:
I think the form’s prop should be className because it’s the class name of the form? I get why you used formClasses now though

Reply:
I want to be clear about what is (and is _not_) going to have the classes applied. Are you saying the `Form` component should rename the property that's passed in? Because I'd be willing to do that (although I think it will increase confusion), but I strongly feel that it should remain explicit about this value being for the form when in multi-purpose components.

I'd be ok renaming it to `formClassName` for better parallelism with `className` but I really don't want to strip off the "form" part before it reaches its final destination.

Reply:
Stupid collapsed non-diff section. I see belatedly that this is already going into the `Form` itself so this is where it could make sense to make it `className`. 

I feel a little weird about using built-in React terms as variables but you're right that this is a reasonable place to do it.

Reply:
I feel like if it’s the component’s class name className is the way to do it? But yeah I like how you named it going into SwatchWithForm